content/docs/components/heading/theming.mdx

```diff
@@ -2,3 +2,160 @@
 id: heading
 scope: theming
 ---
+
+The `Heading` component is a single part component. All of the styling is applied
+directly to the `h2` element.
+
+> To learn more about styling single part components, visit the [Component Style](/docs/styled-system/component-style#styling-single-part-components) page
+## Theming properties
+
+The properties that affect the theming of the `Heading` component are:
+
+- `variant`: The visual variant of the divider.
+- `colorScheme`: The color scheme of the divider.
+- `size`: The size of the divider. Defaults to `xl`.
+
+## Theming utilities
+
+- `defineStyle`: a function used to create style objects.
+- `defineStyleConfig`: a function used to define the style configuration for a single part component.
+
+```js
+import { defineStyle, defineStyleConfig } from '@chakra-ui/react'
+```
+
+## Customizing the default theme
+
+```js
+import { defineStyle, defineStyleConfig } from '@chakra-ui/react'
+
+const custom = defineStyle({
+    color: "yellow.500",
+    border: "0px",
```

Comment:
The Heading doesn't have a border by default, so you don't need to specify it here. It's a better idea to modify the `fontFamily` and/or `fontWeight` instead.